src/config.rs

```diff
@@ -1,11 +1,18 @@
 use clap::{builder::PossibleValue, Parser, ValueEnum};
 use eth2_network_config::Eth2NetworkConfig;
+use std::net::IpAddr;
 use std::str::FromStr;
 use strum::{EnumString, IntoStaticStr};
 
 #[derive(Debug, Clone, Parser)]
 #[command(about = "Ethereum execution engine multiplexer")]
 pub struct Config {
+    /// Listening address for the HTTP server.
+    #[arg(long, value_name = "IP", default_value = "0.0.0.0")]
```

Comment:
I think it would be good to default to localhost for safety, and let users opt-in to 0.0.0.0.

I got bitten by this with openexecution -- I accidentally exposed my EL to the internet.

```suggestion
    #[arg(long, value_name = "IP", default_value = "127.0.0.1")]
```

Reply:
o yeah great point! thanks